mcx/jax.py

```diff
@@ -115,13 +101,13 @@ def progress_bar_scan(func):
         Note that `body_fun` must be looping over a tuple who's first element is `np.arange(num_samples)`.
         This means that `iter_num` is the current iteration number
         """
-        print_rate = int(num_samples / 10)
 
         def wrapper_progress_bar(carry, x):
+            "x is a tuple: (iteration_number, key)"
             iter_num = x[0]
-            iter_num = _progress_bar((iter_num, print_rate), iter_num)
+            _ = _update_progress_bar(iter_num, iter_num)
```

Comment:
`_update_progress_bar` returns `iter_num`, but I set it to `_` to reflect that fact we don't use it again

Reply:
Maybe I missed something, but do you need to return anything at all?